components/quickbooks/actions/create-invoice/create-invoice.mjs

```diff
@@ -1,75 +1,76 @@
-// legacy_hash_id: a_m8ij6m
-import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+import quickbooks from "../../quickbooks.app.mjs";
 
 export default {
   key: "quickbooks-create-invoice",
   name: "Create Invoice",
-  description: "Creates an invoice.",
-  version: "0.1.1",
+  description: "Creates an invoice. [See docs here](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/invoice#create-an-invoice)",
+  version: "0.1.2",
   type: "action",
   props: {
-    quickbooks: {
-      type: "app",
-      app: "quickbooks",
-    },
-    line_items: {
+    quickbooks,
+    lineItems: {
+      label: "Line Items",
       type: "any",
       description: "The minimum line item required for the request is one of the following: \n* `SalesItemLine` type\n* `GroupLine` type\nand Inline subtotal using `DescriptionOnlyLine`",
     },
-    customer_ref_value: {
+    customerRefValue: {
```

Comment:
Can we move `customerRefValue` to the app file? It is used here, and in `create-payment`